docs/group_faqs/gf_faqs.rst

```diff
@@ -1,21 +1,19 @@
 Frequently Answered Questions
 =============================
 
-What are plans in Solace PubSub+ cloud?
+What are service types in Solace PubSub+ cloud?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-On PubSub+ Cloud you could choose a plan to use for your messaging needs, below are the enterprise plans PubSub+ cloud offers:
+On PubSub+ Cloud you could choose a service type to use for your messaging needs, below are the enterprise service types PubSub+ cloud offers:
 
-* Nano (This is shared plan which could be used by developers)
-* Kilo
-* Mega
-* Giga
-* Tera
+* Developer-Shared (Shared, Single-Node, uses Nano service class)
+* Developer (Dedicated, Single-Node, uses Developer service class)
+* Enterprise (Dedicated, High-Availability, uses Kilo, Mega, Giga, Tera depending on client specifications)
```

Comment:
Again here we can remove the service class stuff and say "Dedicated Infrastructure"

Reply:
Instead of the service class stuff put something like "Customizable scale."